probe-rs/src/architecture/arm/core/armv8a.rs

```diff
@@ -1272,6 +1307,14 @@ impl<'probe> MemoryInterface for Armv8a<'probe> {
         Ok(())
     }
 
+    fn write_16(&mut self, address: u64, data: &[u16]) -> Result<(), Error> {
+        for (i, word) in data.iter().enumerate() {
+            self.write_word_16(address + ((i as u64) * 4), *word)?;
```

Comment:
Same as above, what is 4?

Reply:
nice catch, fixed. Turns out `write_8` was wrong and I copypasted from that. Fixed that too.